### PR TITLE
Wallet transactions support

### PR DESCRIPTION
**Pull request by sethlivingston**
- Endpoints and types
- Will add test subfolder in an immediate follow-on PR

### CODE REVIEW

File: .github/workflows/pull-request.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ on:
 jobs:
   changedfiles:
     name: Publish to docs
-    runs-on: [ubuntu-latest]
+    runs-on: [self-hosted, services, X64]
```
Comment (eric-moov) [inline .github/workflows/pull-request.yml:11]: ?????

Comment (sethlivingston) [reply .github/workflows/pull-request.yml:11]: These are the runners all the other FE repos use. I wanted to see if fixing this fixed the docs PR error.

**Review by eric-moov — COMMENTED** [review]
No updates to examples?  Or new examples to create?

**Comment by sethlivingston** [comment]
> No updates to examples? Or new examples to create?

Next PR

File: .github/workflows/pull-request.yml (hunk 1 above)
Comment (adamdecaf) [reply .github/workflows/pull-request.yml:11]: I wouldn't use private runners on an open-source repository. Let's keep `ubuntu-latest`.

Comment (sethlivingston) [reply .github/workflows/pull-request.yml]: We attempt to create a PR in the docs repo when changes are made in this repo. Here is the error: https://github.com/moovfinancial/moov-node/actions/runs/3483437814/jobs/5826924611

Comment (adamdecaf) [reply .github/workflows/pull-request.yml]: GITHUB_TOKEN only has permissions for the repository it's running in and forks have even less access. 

https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token

Comment (adamdecaf) [reply .github/workflows/pull-request.yml]: I'm not sure if giving an oss repository access to write to docs.moov.io is the best idea. Can we run a scheduled job in docs to pull the latest moov-node code?

Comment (sethlivingston) [reply .github/workflows/pull-request.yml]: Yes, we'll explore that and possibly other ways to do this. Thanks, @adamdecaf!

**Pull request merged**